womtool/src/main/scala/womtool/graph/WomGraph.scala

```diff
@@ -60,7 +60,7 @@ class WomGraph(graphName: String, graph: Graph) {
       // Don't include the scatter expression input port here since they're added later in `internalScatterNodesAndLinks`
       // Round up the gathered output ports so it's obvious they're being gathered.
       s"""
-        |${combine((s.inputPorts -- s.scatterCollectionExpressionNode.inputPorts) map portLine)}
+        |${combine((s.inputPorts -- s.scatterCollectionExpressionNodes.head.inputPorts) map portLine)}
```

Comment:
Can we show all of the scatter collection inputs feeding into the scatter?

Reply:
yeah I meant to go back and fix that but forgot

Reply:
wait this was originally removing the scatter expressions input ports from the scatter input ports right ?